src/Kafka/Client.php

```diff
@@ -155,17 +147,17 @@ private function updateStreamOptions()
      */
```

Comment:
No more caching broker list in Client.php, this is now down in the two ClusterMetadata implementations (ZooKeeper and MetadataFromKafka)